contracts/sg-ics721/src/execute.rs

```diff
@@ -0,0 +1,25 @@
+use cosmwasm_std::{to_binary, Binary, Env, StdResult};
+use ics721::{execute::Ics721Execute, token_types::Class};
+
+use crate::state::SgIcs721Contract;
+
+impl Ics721Execute for SgIcs721Contract {
+    fn init_msg(&self, env: &Env, class: &Class) -> StdResult<Binary> {
+        to_binary(&sg721::InstantiateMsg {
+            // Name of the collection MUST be class_id as this is how
+            // we create a map entry on reply.
+            name: class.id.clone().into(),
+            symbol: class.id.clone().into(),
+            minter: env.contract.address.to_string(),
+            collection_info: sg721::CollectionInfo {
+                creator: env.contract.address.to_string(),
+                description: "".to_string(),
+                image: "https://arkprotocol.io".to_string(),
+                external_link: None,
+                explicit_content: None,
+                start_trading_time: None,
+                royalty_info: None,
+            },
```

Comment:
SG team needs to check, what whether this is suffice for listing on marketplace. Can this be changed later, after transferred or does it needs to be defined properly on first NFT transfer to SG, when sg721-base is created?

Reply:
This can be changed later if it's an sg721-updatable. Can you double check @yubrew and @MightOfOaks.

Reply:
Both sg721-base and sg721-updatable support CollectionInfo updates. The only field that can't be updated is the creator.

Reply:
And who can update it? Owner who instantiated sg721? In this case it would be ics721 - which we are not in control of.

Reply:
`collection_info.creator` is the only address that can perform an update currently. Is the actual creator address accessible somehow?